### DISCUSSION TITLE
Can't compile after updating plugins?

### DISCUSSION THREAD

**Discussion by tomkail** [RageAgainstThePixel/com.openai.unity#168] [Q&A] [answered]
I've updated all our RageAgainstThePixel dependencies, but it looks like this package references a version of the Rest plugin that doesn't contain the Debug field in Rest.DownloadAudioClipAsync. It uses version 2.4.6 of the Rest package; latest appears to be 2.5.0. Oddly, it seems that parameter isn't in the latest code for the Rest plugin either?

**Comment by StephenHodgson** [comment, marked as answer]
I have not pushed that update for the rest package to openai package yet. I suggest you roll back to 2.4.6

**Comment by StephenHodgson** [comment]
@tomkail all packages have been updated in the latest release #173
